include/alpaka/api/cuda/computeApi.hpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2025 René Widera
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#pragma once
+
+#include "alpaka/CVec.hpp"
+#include "alpaka/api/unifiedCudaHip/ComputeApi.hpp"
+#include "alpaka/core/config.hpp"
+
```

Comment:
```suggestion

#include <type_traits>

```
To use integral_constant instead of CVec